### PR TITLE
All 3 Homework Done

### PR DESCRIPTION
**Pull request by Semmed-Kosayev**

### CODE REVIEW

File: shammad/src/Homework1.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+        System.out.print("Enter the character: ");
+        String character = scanner.nextLine();
+
+        //Extracting variables
+        String firstHalf = text.substring(0, index);
+        String secondHalf = text.substring(index);
+        String replacingCharacterInString = String.valueOf(text.charAt(index));
+
+        System.out.println(secondHalf.replaceFirst(replacingCharacterInString, character));
```
Comment (sabuhihuseynov) [inline shammad/src/Homework1.java:25]: you need to combine this with firstHalf and then print

File: shammad/src/Homework1.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+        //Taking inputs
+        System.out.print("Enter a text: ");
+        String text = scanner.nextLine();
+
+        System.out.print("Enter the index (starts at 0): ");
+        int index = Integer.parseInt(scanner.nextLine());
+
+        System.out.print("Enter the character: ");
+        String character = scanner.nextLine();
```
Comment (sabuhihuseynov) [inline shammad/src/Homework1.java:18]: you can use char data type instead of String

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
pls look

File: shammad/src/Homework2.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+public class Homework2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        //Taking inputs
+        System.out.print("Enter a text: ");
+        String text = scanner.nextLine();
+
+        System.out.print("Enter the character to see its index: ");
+        String character = scanner.nextLine();
```
Comment (sabuhihuseynov) [inline shammad/src/Homework2.java:13]: here also) I commented in one place pls check whole homework for commented situation

File: shammad/src/Homework3.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        System.out.print("Enter the second number: ");
+        double num2 = scanner.nextDouble();
+
+        System.out.printf("Addition: %.1f\n", num1+num2);
+        System.out.printf("Subtraction: %.1f\n", num1-num2);
+        System.out.printf("Multiplication: %.1f\n", num1*num2);
+        System.out.printf("Division: %.1f\n", num1/num2);
+        System.out.printf("Modulus: %.1f\n", num1%num2);
+
+        //Second part
```
Comment (sabuhihuseynov) [inline shammad/src/Homework3.java:23]: pls use Ctrl + Alt + L

File: shammad/src/Homework2.java (hunk 3 above)
Comment (Semmed-Kosayev) [reply shammad/src/Homework2.java:13]: Will do ✅

File: shammad/src/Homework1.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+public class Homework1 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        //Taking inputs
+        System.out.print("Enter a text: ");
+        String text = scanner.nextLine();
+
+        System.out.print("Enter the index (starts at 0): ");
+        int index = Integer.parseInt(scanner.nextLine());
```
Comment (sabuhihuseynov) [inline shammad/src/Homework1.java:13]: we can use nextInt() instead get string and then parsing it to integer

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
plss)

**Review by sabuhihuseynov — APPROVED** [review]

**Pull request merged**